src/yunohost/certificate.py

```diff
@@ -810,9 +810,20 @@ def _domain_is_accessible_through_HTTP(ip, domain):
 
 
 def _name_self_CA():
-    cert = crypto.load_certificate(
-        crypto.FILETYPE_PEM, open(SELF_CA_FILE).read())
-    return cert.get_subject().CN
+    ca_conf = os.path.join(SSL_DIR, "openssl.ca.cnf")
```

Comment:
maybe we can forget about my strange setup ? ( bit of history: I used to have sefl-signed certificate, with CA on main domain; then change the main domain, then dirty hack to set up a gandi certificate, and then hack with letsencrypt by hand ).